### ISSUE TITLE
[Bug] Library randomly showing filtered results by default

### ISSUE THREAD

**Issue by Mayriad** [Difegue/LANraragi#376]
**LRR Version and OS**

LANraragi for Windows 0.7.5 (MSI installer) on Microsoft Windows 10 build 19042.685. Archives are located on a NAS and accessed via a mount point in the content folder. Please refer to #274 my big list of LRR bugs and suggestions to see more details. Some problems may be caused by my relatively awkward setup.

**Bug Details**

After adding archives, the library can sometimes show filtered results by default and the message would be like this:
```
Showing 1 to 100 of 127 ancient chinese lithographies. (filtered from 138 total entries)
```
The unfiltered count of 138 entries is correct in the example above, but it can be affected by #282 in general. I can view all entries if I do an empty search, and I think restarting LRR occasionally fixed this problem when I tested. This issue happens randomly and I cannot always reproduce it, which means adding the same archives may or may not give this issue. Unlike #282, this does not seem to be affected by the number of archives added at once.

**Comment by Difegue** [comment]
It's not a bug, it's a feature.
<sup><sub>I can't believe I get to say this unironically for the first time</sup></sub>

The library automatically hides entries it can't find on the filesystem.  
This could be exposed to the user better, but basically if you run a database clean the filtered entries should disappear on their own.

**Comment by Mayriad** [comment]
Yep, cleaning the database would remove the filter and show all entries by default. The cleaning process runs significantly slower and says `Successfully cleaned the database and removed 0 entries!` at the end.

If it is a feature then it has bugs, because:
- LRR can indeed find the hidden archives on the filesystem and show them if you check `untagged archives only` or `new archives only`. Previously I said "I can view all entries if I do an empty search" but I could not reproduce that just now.
- The library does automatically hide entries it cannot find, but when I remove entries the filtered and unfiltered numbers in that `Showing 1 to 100 of 127 ancient chinese lithographies. (filtered from 138 total entries)` message will not be updated to reflect the hidden entries.

It is interesting to note that adding a small number of archives at once tends to give this bug, and adding a large number tends to give bug #282.

**Comment by Difegue** [comment]
The total count in the search API now only shows the existing files on the filesystem using the shinobu filemap (fix for #510 ), which should fix this as well.
